src/Microsoft.AspNet.Mvc.Core/Internal/ModelClientValidationRemoteRule.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Mvc.ModelBinding;
+
+namespace Microsoft.AspNet.Mvc.Internal
+{
+    /// <summary>
+    /// <see cref="ModelClientValidationRule"/> containing information for HTML attribute generation in fields a
+    /// <see cref="RemoteAttribute"/> targets.
+    /// </summary>
+    public class ModelClientValidationRemoteRule : ModelClientValidationRule
```

Comment:
`ModelClientValidationRemoteRule` exactly follows the perhaps-unfortunate `ModelClientValidationEqualToRule` pattern.

anyhow, fodder for refactoring model binding.  not a project for today.